### PR TITLE
Introduce and use static backup for network lottie element

### PR DESCRIPTION
**Pull request by jarolrod**
Introduces and puts to use a static backup for the network lottie element which is meant to be shown on the strengthen bitcoin onboarding page. The svg file has been provided by @GBKS. The png was created with inkscape, and run through the [optimize-pngs.py](https://github.com/bitcoin-core/bitcoin-maintainer-tools/blob/master/optimize-pngs.py) script; the output of the script is as follows: 

```
optimizing network-dark.png... done
optimizing network-light.png... done
summary:
+++++++++++++++++
network-dark.png
  size diff from: 34413 to: 25693
  old sha256: 60f8cfd5c65edd4a152086cfcae72b580bf60dafd753929ae65ee509be88d24b
  new sha256: 9be93ea8b2c7df192d031b16817a7c9112ef752a5cf140206431b1bb467e42f5

network-light.png
  size diff from: 41530 to: 26290
  old sha256: b989ba4e59045cbb40cc138395c1285713c866e783cd4e09df152592596d0011
  new sha256: ae3e3ef968d7886bca34e46de04dcdbcb62c66efb81254318c2bf6f93b200e60

completed. Checksum stable: False. Total reduction: 23960 bytes
```

| Light Mode | Dark Mode |
| --------------- | -------------- |
| light | dark |

[Windows](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip?branch=pull/157)
[Intel macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip?branch=pull/157)
[Apple Silicon macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos_arm64/insecure_mac_arm64_gui.zip?branch=pull/157)
[ARM64 Android](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip?branch=pull/157)

### CODE REVIEW

**Comment by jarolrod** [comment]
setting as draft until we have a static backup that works with light mode

**Comment by jarolrod** [comment]
updated from 6cc4e1d to 73d6d2a
changes; Rebased over master, use updated svg elements, allow UI to switch between the light and dark backup

**Comment by jarolrod** [comment]
updated from 73d6d2a to 1060380

changes: fix issue with make file

**Comment by hebasto** [comment]
Rebase?

**Comment by jarolrod** [comment]
updated from 1060380 to 88f9df4

changes: rebased over master

**Review by hebasto — APPROVED** [review]
ACK 88f9df422f18b71f4035c41e8e96cc698af1990a, I have reviewed the code and it looks OK, I agree it can be merged.

File: src/Makefile.qt.include [hunk 1]
```diff
@@ -305,7 +305,9 @@ QML_RES_ICONS = \
   qml/res/icons/cross.png \
   qml/res/icons/export.png \
   qml/res/icons/gear.png \
-  qml/res/icons/info.png
+  qml/res/icons/info.png \
+  qml/res/icons/network-dark.png \
+  qml/res/icons/network-light.png
```
Comment (hebasto) [inline src/Makefile.qt.include:310]: `./test/lint/lint-whitespace.py` complains about a trailing space.

**Pull request merged**